src/test/resources/connections.properties

```diff
@@ -1,4 +1,10 @@
 h2.engine=com.feedzai.commons.sql.abstraction.engine.impl.H2Engine
 h2.jdbc=jdbc:h2:target/pdb
 h2.username=pdb
-h2.password=pdb
\ No newline at end of file
+h2.password=pdb
+
+# Example for oracle
+oracle.engine=com.feedzai.commons.sql.abstraction.engine.impl.OracleEngine
```

Comment:
please remove this from default connections. custom connection.properties should be only local